Find-and-Replace/Find-and-replace-text-within-table/.NET-Framework/Find-and-replace-text-within-table/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace Find_and_replace_text_within_table
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Program program = new Program();  
+            //Load the Input document.
+            using (WordDocument document = new WordDocument(Path.GetFullPath(@"../../Input.docx"), FormatType.Docx))
+            {              
+                //Access table in Word document.
+                WTable table = document.Sections[0].Tables[0] as WTable;
+                program.FindTextWithinTable(table, document); 
```

Comment:
Whether we need to call using class name?

Reply:
No, I have changed the method access and naming to correct this.